vcbuild.bat

```diff
@@ -208,6 +212,16 @@ if "%test%"=="test-simple" set test_args=%test_args% simple
 if "%test%"=="test-message" set test_args=%test_args% message
 if "%test%"=="test-gc" set test_args=%test_args% gc
 if "%test%"=="test-all" set test_args=%test_args%
+if "%test%"=="test-v8" goto test-v8
+if "%test%"=="test-v8-intl" goto test-v8-intl
+
```

Comment:
Good catch.
On Apr 24, 2015 3:47 AM, "orangemocha" <notifications@github.com> wrote:

> In vcbuild.bat
> <https://github.com/joyent/node/pull/14185#discussion_r29039825>:
>
> > @@ -208,6 +212,16 @@ if "%test%"=="test-simple" set test_args=%test_args% simple
> >  if "%test%"=="test-message" set test_args=%test_args% message
> >  if "%test%"=="test-gc" set test_args=%test_args% gc
> >  if "%test%"=="test-all" set test_args=%test_args%
> > +if "%test%"=="test-v8" goto test-v8
> > +if "%test%"=="test-v8-intl" goto test-v8-intl
> > +
>
> Note that the control here was falling through to :build-node-weak before
> your change. Now, it will fall through to :test-v8, causing v8 tests to
> *always* run.
>
> You can either add a goto :build-node-weak after these checks or move
> :test-v8 and :-test-v8-intl somewhere below where they are not in the
> default control flow path, essentially making them subroutines.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joyent/node/pull/14185/files#r29039825>.
>